src/main/kotlin/com/sexysisters/tojserverv2/domain/wiki/policy/UpdateAuthority.kt

```diff
@@ -14,7 +14,7 @@ import org.springframework.stereotype.Component
 @Component
 class UpdateAuthority : WikiPolicy {
     override fun check(student: Student, school: School) {
-        if (student.school!!.code != school.code) {
+        if (student.school.code != school.code) {
```

Comment:
제 생각엔 code 자체를 비교하는 로직은 돌아가지 않을 것 같습니다.
```suggestion
        if (student.school.codeValue() != school.codeValue()) {
```
위 코드처럼 값을 비교하는 것이 좋을 것 같습니다. 만약 code 자체로 비교가 된다면 굳이 바꿀 필요는 없을 것 같아요!

더 고친다면 아래처럼 할 수 있을 것 같습니다!
```kotlin
student.schooCode()
```

Student.kt
```kotlin
fun schoolCode() {
    return school.codeValue()
}
```

Reply:
원시 타입을 래핑한 객체의 getter는 저희끼리 컨벤션으로 `xxxValue()`로 고정하는게 좋을 것 같아요

Reply:
제안하신 코드로 적용하는게 좋을 것 같아요

Reply:
컨벤션을 만드는거 좋은 것 같아요! 저도 앞으로 ```xxxValue()```로 컨벤션을 지키도록 하겠습니다.